meinberlin/apps/plans/assets/PlansList.jsx

```diff
@@ -66,8 +66,9 @@ class LazyBackground extends React.Component {
         className={this.props.isHorizontal ? 'u-lg-only-display maplist-item__img' : 'maplist-item__img'} style={{
           backgroundPosition: 'center',
           backgroundRepeat: 'no-repeat',
-          backgroundImage: this.state.source ? `url(${this.state.source})` : 'none'
-        }} alt=""
+          backgroundImage: this.state.source ? url(this.state.source) : 'none'
```

Comment:
@goapunk i'm bad a syntax, url is not defined? sorry dumb question

Reply:
needs to be `'url(' + this.state.source + ')'` I guess?